### DISCUSSION TITLE
Fake sender name in github notification emails

### DISCUSSION THREAD

**Discussion by andreasvarga** [community/community#136671] [Code Security] [answered]
### Select Topic Area

Question

### Body

My computer is a desktop PC with Windows 11 and I am using Thunderbird for managing my emails (German version).

In the last two weaks I became aware that ALL my emails containing notifications from Github, related to earlier comments or opened issues contain in the header part the following information  (the example is the last received email): 

```
From: Beliavsky

To: MakieOrg/Makie.jl <Makie.jl@noreply.github.com>

CC: Comment <comment@noreply.github.com>, Andreas Varga <varga.andreas@gmail.com>

Reply to: MakieOrg/Makie.jl <reply+ALJDHEEDQWXCYHB26LPHODWE2JIVVEVBNHHCINB76Q@reply.github.com>

Re: [MakieOrg/Makie.jl] Makie Roadmap [WIP] (#519)
```

The name "Beliavsky" appears in all my emails (this is a real Github user name!), but when I am moving the cursor of my mouse on this name, I receive the correct name `Frederic Freyer <notifications@github.com>`, which I suppose I should see (as I checked on my tablet, where I don't observe any such anomaly). 

I wonder what could be the source of this behaviour (perhaps a physhing activity)? Could it be that my PC is affected by a virus? 

Many thanks in advance for any hint how to proceed further. 

Sorry if this message has been posted in the wrong discussion group.

**Comment by andreasvarga** [comment, marked as answer]
I am not quite sure, but I think I have an idea what was wrong with displaying the name "Beliavsky" for all github notifications by my mailer. Since all emails containing github notifications have a sender of the form `user-name <notifications@github.com>`, my mailer (Thunderbird) probably stored in my contacts `beliavsky<notifications@github.com>`, which stays for the all senders with email address `notifications@github.com`. After deleting `beliavsky<notifications@github.com>` from my contacts, everything is displayed as expected.

**Reply by egc112** [reply]
This has plagued me for some time, you solution worked, thanks
